pycosep/community_separability.py

```diff
@@ -327,8 +329,8 @@ def _randomize_communities(communities, total_permutations):
     total_communities = len(communities)
 
     for ix in range(total_permutations):
-        np.random.seed(ix)
-        positions = np.random.permutation(total_communities)
+        np.random.seed(ix+1)            # to equal the Matlab random permutation indices
```

Comment:
Do we want to match MATLAB's seeding for a better debugging experience, or does it impact the results? 🤔

Reply:
Partly for the debugging purpose. I think it is fine to use the original random,  since the fluctuation of the results is in an acceptable range compared to Matlab results.